cardano-crypto-tests/vector-tests/Main.hs

```diff
@@ -0,0 +1,20 @@
+module Main (main) where
+
+import qualified Test.EcdsaSecp256k1Tests as EcdsaSecp256k1Tests
+import qualified Test.SchnorrSecp256k1Tests as SchnorrSecp256k1Tests
+import Test.Tasty (TestTree, defaultMain, testGroup)
+import qualified TestVector.EcdsaSecp256k1Vectors as EcdsaSecp256k1Vectors
+import qualified TestVector.SchnorrSecp256k1Vectors as SchnorrSecp256k1Vectors
+
+main :: IO ()
+main = defaultMain tests
+
+tests :: TestTree
+tests =
+  testGroup
+    "Secp-256k1 tests"
+    [ EcdsaSecp256k1Tests.tests,
+      SchnorrSecp256k1Tests.tests,
```

Comment:
I can see these are unit-tests. Why are we having more unit-tests than those presented in the [DSIGN test framework](https://github.com/input-output-hk/cardano-base/blob/master/cardano-crypto-tests/src/Test/Crypto/DSIGN.hs)? There are duplicate tests, and if there are any missing tests in the DSIGN framework, they should be added there instead of here.

Reply:
Yeah, you are right, all cases written here are covered in Quickcheck. We thought duplicate tests would do no harm but yeah as you mentioned they would also not be useful. Dropping the unit tests in the next update.